packages/backend/db/migrations/20240324205000_add_user_password.sql

```diff
@@ -0,0 +1,17 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE `user_authentication` (
```

Comment:
できればテーブル名は複数形で統一したいです。
https://scrapbox.io/stlatica/%E3%83%87%E3%83%BC%E3%82%BF%E3%83%99%E3%83%BC%E3%82%B9%E8%A6%8F%E5%89%87%E9%A1%9E

`user_auth_credentials`とかでいかがでしょうか？
```suggestion
CREATE TABLE `user_auth_credentials` (
```